modules/ols-thinking-model-generates-delineator-prompt.adoc

```diff
@@ -0,0 +1,13 @@
+// This module is used in the following assemblies:
+// troubleshoot/ols-troubleshooting-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ols-thinking-model-generates-delineator-prompt_{context}"]
+= Thinking model generates delineator prompt
+
+In some cases, thinking or reasoning models produce output that has delineators, such as `THOUGHT` or `reasoning`, to separate the thinking process from the model output that the {ols-long} Service uses to answer your question. 
+
+The {ols-long} Service does not configure the delineator behavior or produce the delineators in the output. The delineator feature is a model configuration setting. Typically, you can disable the feature using one of the following methods:
+
+* Append your question prompt with the specific keyword that the model requires to toggle the delineator feature off. For example, `/nothink`.
+* Use API configuration settings in the model to turn the delineator feature off  
```

Comment:
I think this should be reworded to say that the _inference server_ configuration may need to be modified to disable this, and that would be in the docs for the inference server and/or the model.

Reply:
done